docs/source/examples/combine_ddd_zne.md

```diff
@@ -0,0 +1,241 @@
+
+
+We will explain this further in the noise model section, but suffice it to say that these are intended to amplify the effect of time correlated noise. 
+
+```{code-cell}
+def idle_qubits(circuit, qubits, idle_steps):
+    """Set qubits to idle for specfied number of steps 
+    in by inserting Identity gates in each `moment`
+    """
+    for step in range(idle_steps):
+        circuit.append(cirq.I(q) for q in qubits)
+
+    return circuit
+```
+
+```{code-cell}
+def ghz(num_qubits, idle_steps=0):
+    # Create  qubit registers
+    qubits = cirq.LineQubit.range(num_qubits)
+
+    # Create a quantum circuit
+    circuit = cirq.Circuit()
+    
+    # Add CNOT gates to entangle the first qubit with each of the other qubits
+    for i in range(num_qubits):
+        if i == 0: 
+            # Add a Hadamard gate to the first qubit
+            circuit.append(cirq.H(qubits[0]))
+        else:
+            circuit.append(cirq.CNOT(qubits[0], qubits[i]))
+            # Set qubits to idle for specfied number of steps
+            # in the form of Identity gates
+            other_qubits = qubits[1:i] + qubits[i+1:]
+            circuit = idle_qubits(circuit, other_qubits, idle_steps)
+    
+    return circuit
+```
+
+### Defining the circuit
+
+For this example we will create a 6 qubit GHZ state with two idle steps after each moment:
+
+```{code-cell}
+num_qubits = 6
+circuit = ghz(num_qubits, idle_steps=3)
+
+print(circuit)
+```
+
+In the diagram above, the horizontal brakets above and below specific gates represent the [cirq](https://quantumai.google/cirq) notion of [moments](https://quantumai.google/reference/python/cirq/Moment), which are time-slices intended to help with qubit scheduling (e.g. making sure qubits arrive at a gate at the same time). 
+
+In the our noise model, we will apply errors after each moment. Adding the additional identity gates within each "moment" should amplify the effect of these time-correlated errors.
+
+**Note:** Due to cirq's definition of moments, the first two gates of the circuit (`cirq.H(qubits[0])` and `cirq.CNOT(qubits[0], qubits[i])`) are in the same moment. This means the idle steps cannot be added between these gates, and therefore the time correlated noise will have less of an effect for the start of the circuit. 
+
++++
+
+## Noise model and executor
+
+**Importantly**, since DDD is designed to mitigate time-correlated (non-Markovian) noise, we simulate systematic $R_z$ rotations and depolarising noise applied to each qubit after each time step. This corresponds to noise which is strongly time-correlated and, therefore, likely to be mitigated by DDD. We 
```

Comment:
`We` hanging at the end of this paragraph.

Reply:
Deleted